docs/ase-file-specs.md

```diff
@@ -338,13 +340,51 @@ there is an User Data Chunk at the first frame after the Palette Chunk.
     DWORD       Flags
                   1 = Has text
                   2 = Has color
+                  4 = Has properties
     + If flags have bit 1
       STRING    Text
     + If flags have bit 2
       BYTE      Color Red (0-255)
       BYTE      Color Green (0-255)
       BYTE      Color Blue (0-255)
       BYTE      Color Alpha (0-255)
+    + If flags have bit 4
+      DWORD     Number of properties maps
+      + For each properties map:
+        DWORD     Properties maps key (Zero or extension's Entry ID (see External Files Chunk))
+        DWORD     Number of properties
+        + For each property:
+          STRING    Property name
+          WORD      Property type
+          + If property type is bool (0x0001)
+            BYTE    False (0x00) or true (any value different than 0x00)
+          + If property type is unsigned char (0x0002)
+            BYTE
+          + If property type is short (0x0003)
+            SHORT
+          + If property type is unsigned short (0x0004)
+            WORD
+          + If property type is int (0x0005)
+            LONG
+          + If property type is unsigned int (0x0006)
+            DWORD
+          + If property type is long long (0x0007)
+            LONG64
+          + If property type is unsigned long long (0x0008)
+            QWORD
+          + If property type is fixmath::fixed (0x0009)
```

Comment:
`fixmath::fixed` is not a public data (not needed to specify this internal type in this doc). We just reference fixed point numbers as FIXED.